ambari-logsearch/ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/conf/SecurityConfig.java

```diff
@@ -100,6 +100,9 @@
   @Inject
   private LogSearchConfigApiConfig logSearchConfigApiConfig;
 
+  @Inject
+  private LogsearchAuthenticationProvider logsearchAuthenticationProvider;
+
```

Comment:
Is thread-safety a concern here?

Reply:
Before this patch there were more LogsearchAuthenticationProvider instances because of the incorrect spring configuration + it was instantiated via the new operator but only one instance was used.